packages/runtime/src/runtimes/react/controls/control.tsx

```diff
@@ -47,6 +48,20 @@ type ControlValueProps = {
   control?: InstanceType<ControlDefinition>
 }
 
+type Result<T> = { success: true; value: T } | { success: false; error: string }
+
+export function resolveValue(
+  data: DataType<ControlDefinition> | undefined,
+  definition: ControlDefinition,
+): Result<ResolvedValueType<ControlDefinition>> {
+  try {
+    return { success: true, value: definition.resolveValue(data).readStableValue() }
+  } catch (error) {
+    console.error(`Error resolving '${definition.controlType}' prop value`, { error, data })
+    return { success: false, error: `${error}` }
```

Comment:
Will this interpolated error properly stringify?

Reply:
I think so:

<img width="250" alt="error" src="https://github.com/user-attachments/assets/ca8bd97b-835b-42bb-a748-dec285c0d42c">